pkg/api/dtos/plugins.go

```diff
@@ -45,6 +45,7 @@ type PluginListItem struct {
 	SignatureType plugins.SignatureType   `json:"signatureType"`
 	SignatureOrg  string                  `json:"signatureOrg"`
 	AccessControl accesscontrol.Metadata  `json:"accessControl,omitempty"`
+	Angular       bool                    `json:"angular"`
```

Comment:
```suggestion
	AngularDetected       bool                    `json:"angularDetected"`
```

Just to make it more clear what this prop means